service/history/transferQueueStandbyTaskExecutor.go

```diff
@@ -251,7 +251,7 @@ func (t *transferQueueStandbyTaskExecutor) processCloseExecution(
 
 		// DO NOT REPLY TO PARENT
 		// since event replication should be done by active cluster
-		return nil, t.recordWorkflowClosed(
+		return nil, t.archiveVisibility(
```

Comment:
This will conflict with Yichao's change https://github.com/temporalio/temporal/pull/2806

Reply:
It should be fine... I am not changing the logic inside recordWorkflowClosed.

Reply:
It is just a rename to better reflect what this method is doing.